demo/mei_dj_free_demo.ipynb

```diff
@@ -0,0 +1,680 @@
+{
```

Comment:
<p>Move up to beginning</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/mei/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>